src/index.html

```diff
@@ -7,6 +7,419 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo"
+        >
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__item is-active">
+            <a class="is-active" href="#">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a>
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a>
+              Smartphones
+            </a>
+          </li>
+
+          <li data-qa="nav-hover" class="nav__item">
+            <a>
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a>
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a>
```

Comment:
why "a" tag without href?

Reply:
since we don't need to have functionality on it. No problem, it is there now